dev/io.openliberty.org.jboss.resteasy.common/src/org/jboss/resteasy/specimpl/ResteasyHttpHeaders.java

```diff
@@ -177,21 +181,22 @@ public List<Locale> getAcceptableLanguages()
    {
       List<String> vals = requestHeaders.get(ACCEPT_LANGUAGE);
       if (vals == null || vals.isEmpty()
-                      || (vals.size() == 1 && "".equals(vals.get(0).trim()))) { // liberty change
-         return Collections.singletonList(Locale.forLanguageTag("*"));
-      }
-      List<WeightedLanguage> languages = new ArrayList<WeightedLanguage>();
-      for (String v : vals) {
-         StringTokenizer tokenizer = new StringTokenizer(v, ",");
-         while (tokenizer.hasMoreElements()) {
-            String item = tokenizer.nextToken().trim();
-            languages.add(WeightedLanguage.parse(item));
-         }
+                      || (vals.size() == 1 && "".equals(vals.get(0).trim()))) { // Liberty change
+          return LANGUAGE_WILDCARD;
+      } else {
```

Comment:
No reason to add the else here.  Should leave it the way it was.

Reply:
I know it serves no purpose - was just making it consistent with getAcceptableMediaTypes right above it :) 

So it feels like I should remove the 'else' from both, or leave it in both, just aesthetically ...

Reply:
Since it is an overlay we try not to modify more than we need to so we can easily see what is liberty changed parts of the class.